library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt

```diff
@@ -7,9 +7,10 @@ import android.graphics.DashPathEffect
 import android.graphics.Paint
 import android.graphics.Path
 import android.graphics.PathMeasure
+import kotlin.math.ceil
 
 internal class DonutProgressLine(
-    val category: String,
+    val name: String,
     _radius: Float,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/3f8afbf9c8b622c5d946604681947fabe1098940/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressLine.kt#L14"></span>Constructor parameter names should match the pattern: [a-z][A-Za-z\d]*</td>
    </tr>
  </tbody>
</table>